test_notebooks.py

```diff
@@ -57,7 +57,7 @@ def _repo_path():
     name="notebook_filename",
 )
 def _notebook_filename(request):
-    return request.param
+    return os.path.normpath(request.param)
```

Comment:
Thanks!
Is this change intentional?

Reply:
Oh definitely not. I think I did something wrong when merging main to my branch az-pre-commit

Reply:
> Should I change it back with new commit?

I changed it back.